dom-manipulation/main.js

```diff
@@ -0,0 +1,27 @@
+let counter = 0;
+
+function incrementCount(number) {
```

Comment:
in this case `number` would be the event object. I would recommend naming it something more like `e` or `event` to make it more human readable.